src/AppInstallerCLICore/ExecutionContext.cpp

```diff
@@ -487,6 +487,24 @@ namespace AppInstaller::CLI::Execution
     }
 #endif
 
+    void ContextEnumBasedVariantMapActionCallback(const void* map, Data data, EnumBasedVariantMapAction action)
+    {
+        switch (action)
+        {
+        case EnumBasedVariantMapAction::Add:
+            AICLI_LOG(Workflow, Info, << "Setting data item: " << data);
+            break;
+        case EnumBasedVariantMapAction::Contains:
+            AICLI_LOG(Workflow, Info, << "Checking data item: " << data);
+            break;
+        case EnumBasedVariantMapAction::Get:
+            AICLI_LOG(Workflow, Info, << "Getting data item: " << data);
+            break;
+        }
+
+        UNREFERENCED_PARAMETER(map);

```

Comment:
Do we plan to use this variable in the future?

Reply:
I had considered a much more complex runtime checking with tasks actually having information on which data they produced/consumed, but realized that a runtime check is only as good as the tests and so provided very little value.  I didn't feel like removing the parameter at that point.